source/native/ios/Podfile

```diff
@@ -28,7 +28,7 @@ target 'Stargazer' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable the next line.
-  use_flipper!()
+  # use_flipper!()
```

Comment:
Why was flipper disabled?

Reply:
Oops, I was doing some tests and I forget to remove the comment. I'll add flipper again.